tools/nxstyle.c

```diff
@@ -135,67 +211,98 @@ int main(int argc, char **argv, char **envp)
   int externc_lineno;   /* Last line where 'extern "C"' declared */
   int linelen;          /* Length of the line */
   int maxline;          /* Lines longer that this generate warnings */
+  bool silent;          /* Used with go not go test option */
   int n;
   int i;
+  int c;
+  extern char *optarg;
+  extern int optopt;
+  extern int optind;
 
-  maxline  = 78;
-  filename = argv[1];
+  g_gonogo    = false;
+  maxline     = 78;
+  filename    = argv[1];
+  silent      = false;
 
-  /* Usage:  nxstyle [-m <maxline>] <filename>
-   *         nxstyle -h
-   */
-
-  if (argc == 4)
+  if (argc < 2)
```

Comment:
remove if (...){...}, the check at line 270 is enough

Reply:
Done!